### PR TITLE
Fixed Replacing User ID to Username

### PR DESCRIPTION
**Pull request by Rodney-RWR**
I believe I have fixed this, haven't tested yet, don't accept PR yet until I test.

### CODE REVIEW

File: discord.js [hunk 1]
```diff
@@ -84,14 +84,21 @@ client.on('message', msg => {
...
-	// 	replace
-	// })
-
-	// TODO: from DiscordMessageObject(text="hello <@00000000000000001>") it should return String("hello @Username")
-	return message
+  var mentions = [];
+  message.mentions.users.forEach(user => {
+    mentions.push({"userId": user.id, "username": user.username, "discriminator": user.discriminator});
+  });
+  var messageContent = message.content;
```
Comment (VeryBigSad) [inline discord.js:91]: Isn't using `let` instead of `var` is the standard right now?

File: commands.json [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
-    "command_name": [
-      "help",
+    "command_name": [ // the commands name, you can have multiple
+      "help", // will respond to (prefix)help
       "h",
       "?"
     ],
-    "call": "This is your help command. (fill in in commands.json)"
+    "call": "This is your help command. (fill in in commands.json)" 
+    // What the bot responds with, can only be a message as of now, direct message/command support soon (maybe)
```
Comment (VeryBigSad) [inline commands.json:9]: I think comments in JSON are not allowed, we should put this somewhere in `readme.md`

**Review by VeryBigSad — CHANGES_REQUESTED** [review]
amazing! Things to change for this to be ideal:
Just change `var`s to `let`s (not in the entire project, just the commits)
Remove comments from the .json file, I'm somewhat certain JSON doesn't support them

File: commands.json (hunk 2 above)
Comment (Rodney-RWR) [reply commands.json:9]: Fair enough, i think it semi supports comments but it looks like "_comment": "bla bla" I will put the info on how to make them in readme

File: discord.js (hunk 1 above)
Comment (Rodney-RWR) [reply discord.js:91]: Yep, mb, webstorm made it var for some reason.

**Review by Rodney-RWR — COMMENTED** [review]
Alright Should all be good now, still haven't tested will do when I'm home I will add another comment when home, I'm fairly certain it will work though.

**Review by VeryBigSad — APPROVED** [review]

**Pull request merged**